docs/tutorials/flow-web3-app.mdx

```diff
@@ -0,0 +1,453 @@
+> **Note**: This page will walk you through a very bare bones project to get started building a web3 dapp using the Flow Client Library (FCL). If you are looking for a clonable repo, Flow community members have created quickstart templates for different JavaScript frameworks (e.g. [Next.js](https://github.com/muttoni/fcl-nextjs-quickstart), [SvelteKit](https://github.com/muttoni/fcl-sveltekit-quickstart), [Nuxt](https://github.com/bluesign/nuxt3-fcl)). You can consult the complete list [here](https://github.com/ph0ph0/Get-The-Flow-Down#fcl).
+
+## Video Walkthrough
+
+Below is a video walkthrough covering the contents of this tutorial. You are welcome to watch or skip the video walkthrough depending on your style of learning.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/uEoh9SnjqCk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+In this tutorial, you will learn how to interact with an existing smart contract on Flow's testnet, the Profile Contract. Using this contract, you will create a new profile and edit the profile information, both via a wallet.
+
+While this tutorial will use Cadence (Flow's smart contract language), you do not need to know it. Instead, we recommend diving into learning the Cadence language later once you've passed the core FCL concepts down.
+
+## Prerequisite
+- Completed flow quick start guide
```

Comment:
Please be mindful of punctuation / capitalization / grammar. e.g. this should be "Completed the Flow quick start guide" 

also, link to the previous part, otherwise it will be hard for people to find it